src/target/adi.c

```diff
@@ -103,6 +103,7 @@ static const arm_coresight_component_s arm_component_lut[] = {
 	{0x4c4, 0x00, 0, aa_nosupport, cidc_unknown, ARM_COMPONENT_STR("Cortex-M4 ROM", "(Cortex-M4 ROM)")},
 	{0x4c7, 0x00, 0, aa_nosupport, cidc_unknown, ARM_COMPONENT_STR("Cortex-M7 PPB", "(Cortex-M7 PPB ROM Table)")},
 	{0x4c8, 0x00, 0, aa_nosupport, cidc_unknown, ARM_COMPONENT_STR("Cortex-M7 ROM", "(Cortex-M7 ROM)")},
+	{0x4e4, 0x00, 0x0af7, aa_nosupport, cidc_dc, ARM_COMPONENT_STR("CoreSight ROM", "(ROM Table)")},
```

Comment:
It still does show up in my logs :thinking:

Reply:
Ahh, just spotted why too - hmm 🤔 actually, thinking about it we won't include a review note for why because it probably actually makes some sense to display that we found a ROM Table like any other component.. there might need to be a consistency change for ADIv6's version of the parser too then to make it make more sense.